style.css

```diff
@@ -0,0 +1,143 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: content-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  /* 62.5% of 16px = 10px */
+  font-size: 62.5%;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+}
+
+.icons-socialMedia {
+  height: 25px;
+  width: 170.1666717529297px;
+  left: -85px;
+  top: -11px;
+  border-radius: 0;
+  list-style: none;
+  flex-wrap: wrap-reverse;
+  justify-content: flex-start;
+  color: #505f79;
+}
+
+.flex-row {
+  display: flex;
+}
+
+.header {
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  height: 3rem;
+  margin-top: 1.6rem;
+}
+
+.menu-button {
+  margin-top: 0.6rem;
+  margin-left: 2rem;
+  font-style: bold;
+  font-size: 1rem;
+  color: #6070ff;
+  line-height: 1.2rem;
+  line-height: 74%;
+  vertical-align: center;
+  border: 0;
+  background-color: #fff;
+}
+
+.menu-button a {
+  font-family: 'Poppins', sans-serif;
+  font-size: 21px;
+  font-weight: 700;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #6070ff;
+  font-style: bold;
+  vertical-align: center;
+}
+
+.menu-mobile {
+  margin-top: 0.6rem;
+  margin-left: 1.9rem;
+  margin-right: 0.9rem;
+  background-color: #fff;
+}
+
+.main {
+  background-image: url(./img/header-background-mobile.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  padding-bottom: 10rem;
+}
+
+.headline {
+  text-align: left;
+  margin: 0 8%;
+}
+
+.presentation {
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  left: -163px;
+  top: -76px;
+  border-radius: nullpx;
+  font-family: 'Poppins', sans-serif;
+  font-size: 30px;
+  font-weight: 700;
+  text-align: left;
+  color: #172b4d;
+  line-height: 3.2rem;
+  padding-top: 10rem;
+}
```

Comment:
- [ ] [OPTIONAL] Great work implementing the changes in your `h1` from the headline section 💯. However, please kindly consider increasing the size of your font up to 40px and changing your line-height from 3.2rem to 1.2 (without rem, em, or px, just 1.2)  🤓. These changes will make your design more similar to the Figma design. 🔧

Reply:
Ok, thank you.